core_impl/src/main/java/io/opencensus/implcore/stats/MeasureToViewMap.java

```diff
@@ -98,7 +111,12 @@ synchronized void record(TagContext tags, MeasureMap stats, Timestamp timestamp)
     Iterator<Measurement> iterator = stats.iterator();
     while (iterator.hasNext()) {
       Measurement measurement = iterator.next();
-      Collection<MutableViewData> views = mutableMap.get(measurement.getMeasure().getName());
+      Measure measure = measurement.getMeasure();
+      if (!measure.equals(registeredMeasures.get(measure.getName()))) {
+        // measures with a duplicated name will be ignored.
```

Comment:
I think "unregistered measures will be ignored." would be more accurate, since this expression is also true when no registered views contain the measure.

Reply:
Sounds good, updated.